packages/fuels-core/src/abi_encoder.rs

```diff
@@ -104,7 +104,7 @@ impl ABIEncoder {
 
         self.encode_discriminant(*discriminant);
 
-        // The sway compiler has an optimization for enums which have only Units as variants -- such
+        // The high-level language compiler has an optimization for enums which have only Units as variants -- such
```

Comment:
Consistency

Reply:
Good catch! That was me changing stuff and not updating everywhere...